tests/test_sshfsmount.cpp

```diff
@@ -74,6 +124,24 @@ struct SshfsMount : public mp::test::SftpServerTest
         return channel_read;
     }
 
+    void test_command_execution(const command_vector& commands, mp::optional<std::string> target = mp::nullopt)
+    {
+        bool invoked{false};
+        std::string output;
+        auto remaining = output.size();
+        command_vector::const_iterator next_expected_cmd = commands.begin();
+
+        auto channel_read = make_channel_read_return(output, remaining, invoked);
+        REPLACE(ssh_channel_read_timeout, channel_read);
+
+        auto request_exec = make_exec_to_check_commands(commands, remaining, next_expected_cmd, output, invoked);
+        REPLACE(ssh_channel_request_exec, request_exec);
+
+        make_sshfsmount(target.value_or(default_target));
+
+        EXPECT_EQ(next_expected_cmd, commands.end()) << "\"" << next_expected_cmd->first << "\" is not executed in the right place";
```

Comment:
If this doesn't pass, we know the command did not execute, no? Also, `EXPECT_EQ` is unnecessarily verbose here:
```
/home/michal/dev/canonical/multipass/multipass/tests/test_sshfsmount.cpp:146: Failure
Expected equality of these values:
  next_expected_cmd
    Which is: 8-byte object <10-93 30-F7 E3-55 00-00>
  commands.end()
    Which is: 8-byte object <10-94 30-F7 E3-55 00-00>
```

How about this?
```suggestion
        EXPECT_TRUE(next_expected_cmd == commands.end()) << "\"" << next_expected_cmd->first << "\" not executed";
```

Reply:
Better. Changed.